internal/pkg/cli/app_init.go

```diff
@@ -161,10 +166,15 @@ func (o *initAppOpts) Execute() error {
 }
 
 func (o *initAppOpts) createManifest() (string, error) {
-	manifest, err := manifest.CreateApp(o.AppName, o.AppType, o.DockerfilePath)
-	if err != nil {
-		return "", fmt.Errorf("generate a manifest: %w", err)
+	props := &manifest.LBFargateManifestProps{
+		AppName:    o.AppName,
+		Dockerfile: o.DockerfilePath,
+	}
+	props.Path = "/" + o.AppName
+	if o.proj.RequiresDNSDelegation() {
+		props.Path = "*"
```

Comment:
Why? I think the path should be blank and we shouldn't generate path based rules if they're using their own domain. (sorry for the bad name of RequiresDNSDelegation -_-)